### PR TITLE
remove note

### PR DESCRIPTION
**Pull request by get-itips**
confirmed with PM and in lab that at least one of the parameters (EnablePrivacyMode) has effect in Teams

https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9104

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsPrivacyConfiguration.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Modifies an existing set of privacy configuration settings.
 Privacy configuration settings help determine how much information users make available to other users.
 This cmdlet was introduced in Lync Server 2010.
 
-Note: This cmdlet has been deprecated in Microsoft Teams and has no impact.
+
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsPrivacyConfiguration.md:21]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

**Review by dariomws — COMMENTED** [review]
Does anything else in this cmdlet work for Teams?

If not, maybe we should change the note to "Only EnablePrivacyMode for federated users works for Teams".

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Comment by get-itips** [comment]
> Does anything else in this cmdlet work for Teams?
> 
> If not, maybe we should change the note to "Only EnablePrivacyMode for federated users works for Teams".

waiting on some confirmations for now i would just remove the note

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**